motor/providers/vcd/provider.go

```diff
@@ -0,0 +1,115 @@
+package vcd
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/cockroachdb/errors"
+	"github.com/rs/zerolog/log"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"go.mondoo.com/cnquery/motor/providers"
+	"go.mondoo.com/cnquery/motor/vault"
+)
+
+var (
+	_ providers.Instance           = (*Provider)(nil)
+	_ providers.PlatformIdentifier = (*Provider)(nil)
+)
+
+type vcdConfig struct {
+	User     string
+	Password string
+	Host     string
+	Org      string
+	Insecure bool
```

Comment:
Where is this set? Is this the global flag finding it's way to this config via `pCfg`?

Reply:
This happens in line 53, where we set insecure. It is indeed a global flag so that is being passed in when set by the user.